src/codemap_details/CSS/WhyCodeMap.module.css

```diff
@@ -0,0 +1,97 @@
+html,
+body {
+  height: 100%;
+  width: 100%;
+  font-family: Arial;
+}
+
+.Heading {
+  letter-spacing: 0px;
+  font-size: 2.3rem;
+  font-weight: 390;
+  font-family: Arial;
+  text-align: center;
+  margin-top: 10%;
+  max-width: 100%;
+}
+
+.Codemap {
+  margin-top: 10%;
+  height: 300px;
+  width: 100%;
+  background-color: #f2f2f2;
+  border: none;
+  margin: 3%;
+}
+
+.box {
+  margin-top: 4%;
+  transform: translateX(-41%);
+}
+
+.content {
+  transform: translateY(-470%);
+  margin-left: 25%;
+}
+
+.line1 {
+  top: 2767px;
+  width: 900px;
+  margin-top: 60px;
+  border: 1px solid #707070;
```

Comment:
![Capture](https://user-images.githubusercontent.com/32364768/90118571-0f40b600-dd76-11ea-963e-60f7bfaaeb4a.PNG)

@Meghag5 keep the border width as 0.5px. 1px makes it look like a rectangle.